google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -151,7 +153,8 @@ class TracingMessageBatch : public MessageBatch {
       message_spans.swap(message_spans_);
     }
 
-    auto batch_sink_spans = MakeBatchSinkSpans(std::move(message_spans));
+    auto batch_sink_spans =
+        MakeBatchSinkSpans(std::move(message_spans), std::move(options_));
```

Comment:
It is strange to move from a member variable (at least not within an rvalue ref-qualified function).

Reply:
Ack, done.